spring-integration-file/src/main/java/org/springframework/integration/file/filters/CompositeFileListFilter.java

```diff
@@ -42,7 +42,7 @@
  */
 public class CompositeFileListFilter<F> implements ReversibleFileListFilter<F>, ResettableFileListFilter<F>, Closeable {
 
-	private final Set<FileListFilter<F>> fileFilters;
+	protected final Set<FileListFilter<F>> fileFilters;
```

Comment:
Thanks.
I have never understand that rule.
How can we mutate the property if it is `final`.
That's why I made it `protected` instead of getter.